crocks/Pred.js

```diff
@@ -37,7 +40,7 @@ function Pred(runWith) {
       throw new TypeError('Pred.concat: Pred required')
     }
 
-    return Pred(x => runWith(x) && m.runWith(x))
+    return Pred(x => !!runWith(x) && !!m.runWith(x))
```

Comment:
Actually, no need to do the `!!` here. That is just extra, as we already do it in `runWith`